aspnetcore/tutorials/signalr.md

```diff
@@ -2,7 +2,7 @@
 title: Get started with ASP.NET Core SignalR
 author: bradygaster
 description: In this tutorial, you create a chat app that uses ASP.NET Core SignalR.
-ms.author: bradyg
+ms.author: wpickett
```

Comment:
```suggestion
ms.author: wpickett
<!--ms.author: bradyg-->
```

Reply:
@Rick-Anderson,  this is in draft, give me few minutes.  I will re-ask for a review when it is ready.